torch/csrc/jit/OVERVIEW.md

```diff
@@ -1296,27 +1296,71 @@ As a more involved example, the following TorchScript snippet:
 ```python
 @torch.jit.script
 def foo(a : Tensor, b : Tensor):
-        c = 2 * b
+  c = 2 * b
+  d = c + 1
   a += 1
-  if a.max() > 4:
+  if a.item() > 4:
     r = a[0]
   else:
     r = b[0]
-  return c, r
+  ret = [d, r]
+  return ret
+```
+Will produce the following IR:
+```
+graph(%a.1 : Tensor,
+      %b.1 : Tensor):
+  %2 : int = prim::Constant[value=2]() # test/elias.py:6:6
+  %6 : int = prim::Constant[value=1]() # test/elias.py:7:10
+  %14 : int = prim::Constant[value=4]() # test/elias.py:9:16
+  %17 : int = prim::Constant[value=0]() # test/elias.py:10:10
+  %c.1 : Tensor = aten::mul(%b.1, %2) # <string>:3:9
+  %d.1 : Tensor = aten::add(%c.1, %6, %6) # test/elias.py:7:6
+  %a.3 : Tensor = aten::add_(%a.1, %6, %6) # test/elias.py:8:2
+  %13 : Scalar = aten::item(%a.3) # test/elias.py:9:5
+  %15 : bool = aten::gt(%13, %14) # test/elias.py:9:5
+  %r : Tensor = prim::If(%15) # test/elias.py:9:2
+    block0():
+      %r.1 : Tensor = aten::select(%a.3, %17, %17) # test/elias.py:10:8
+      -> (%r.1)
+    block1():
+      %r.2 : Tensor = aten::select(%b.1, %17, %17) # test/elias.py:12:8
+      -> (%r.2)
+  %ret.1 : Tensor[] = prim::ListConstruct(%d.1, %r)
+  return (%ret.1)
 ```
-Will produce a graph like this:
 
-![AliasTracker graph](https://github.com/pytorch/pytorch/blob/master/docs/source/_static/img/aliastracker_graph.png)
+And the following AliasDb (visualized with `AliasDb::toGraphViz`)
+
+![AliasTracker graph1](https://github.com/pytorch/pytorch/blob/master/docs/source/_static/img/aliastracker_graph1.png)
```

Comment:
I don't see the pic but this might be on me

Reply:
They're included in the PR, i dont think the links will work until the PR lands.